doctrine/common-toolbox.metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+    "registrar": [
+        {
+            "language": "php",
```

Comment:
I think that we should use different provider here, like 'doctrine.entities' but it not supported now, //cc @Haehnchen